doc/connectivity-semantics-and-api.md

```diff
@@ -28,7 +28,10 @@ through TLS handshake (or equivalent) and all subsequent attempt to communicate
 have succeeded (or are pending without any known failure ).
 
 TRANSIENT_FAILURE: There has been some transient failure (such as a TCP 3-way
-handshake timing out or a socket error). Channels in this state will eventually
+handshake timing out or a socket error). New RPCs should, by default, fail
+immediately with status UNAVAILABLE, but implementations may support a per-RPC
+option specifying an RPC is 'wait-for-ready' which will leave the RPC as
+pending. Channels in this state will eventually
```

Comment:
...until either the channel becomes READY and the RPC can be attempted or the RPC's deadline expires.

Reply:
FYI: this is pre-existing text.

> until either the channel becomes READY

Yes, but no. CONNECTING can lead to READY, yes. But this is a one-time deal. I say nothing about after the transition to CONNECTING.

> or the RPC's deadline expires

Actually, no. It has nothing to do with whether there are RPCs. We'll still reconnect even without RPCs. The only exception is IDLE_TIMEOUT, which applies to CONNECTING, not TRANSIENT_FAILURE.

Reply:
How about side-stepping this issue entirely by replacing the text starting from "but implementations may" through the end of the sentence with the following:

but implementations may support a `wait_for_ready` option to modify this behavior.

Note that the "wait_for_ready" text should be a link to wait-for-ready.md.